controller/authController.go

```diff
@@ -0,0 +1,160 @@
+package controller
+
+import (
+	"github.com/dgrijalva/jwt-go"
+	"github.com/gofiber/fiber/v2"
+	"golang.org/x/crypto/bcrypt"
+	"mocklogin/database"
+	"mocklogin/model"
+	"strconv"
+	"time"
+)
+
+const SecretKey = "secret"
+
+func Register(context *fiber.Ctx) error {
+	// context has our request data
+
+	// extract the http request data
+	var data map[string]string // declare var data of type map (key:string, value:string)
+
+	// handle error if any
+	if err := context.BodyParser(&data); err != nil {
+		return err
+	}
+
+	// hash the password before inserting into the database
+	password, _ := bcrypt.GenerateFromPassword([]byte(data["password"]), 14)
+
+	// initialize User object with implicit type declaration
+	user := model.User{
+		Name: data["name"],
+		Email: data["email"],
+		Password: password,
+	}
+
```

Comment:
Here you can add 
```suggestion
user.ID = uuid.New().String()
```

as or just [NewString()](https://pkg.go.dev/github.com/google/uuid#NewString)